src/main/java/org/lobid/resources/RdfModel2ElasticsearchEtikettJsonLd.java

```diff
@@ -87,33 +87,46 @@ public RdfModel2ElasticsearchEtikettJsonLd(final Object jsonLdContext) {
 	@Override
 	public void process(final Model originModel) {
 		mainNodeId = null;
-		splitModel2ItemAndResourceModel(originModel);
+		extractItemFromResourceModel(originModel);
 	}
 
-	private void splitModel2ItemAndResourceModel(final Model originalModel) {
+	private void extractItemFromResourceModel(final Model originalModel) {
 		Model copyOfOriginalModel =
 				ModelFactory.createModelForGraph(originalModel.getGraph());
 		final ResIterator subjectsIterator = originalModel.listSubjects();
 		// iterate through all nodes
 		while (subjectsIterator.hasNext()) {
 			final Resource subjectResource = subjectsIterator.next();
 			Model submodel = ModelFactory.createDefaultModel();
-			// only extract sub nodes we don't want to keep in the main model
+			// extract sub nodes (items) => will become documents on their own
 			if (subjectResource.isURIResource()) {
 				if (subjectResource.getURI().startsWith(LOBID_ITEM_URI_PREFIX)) {
 					submodel = extractSubmodel(submodel, subjectResource);
 					toJson(submodel,
 							subjectResource.getURI().toString().replaceAll("#!$", ""));
-					// remove the newly created sub model from the main node
-					copyOfOriginalModel.remove(submodel);
+					// remove some properties from that item which clings to the main node
+					removeProperty(copyOfOriginalModel, subjectResource, "itemOf");
+					removeProperty(copyOfOriginalModel, subjectResource, "describedby");
```

Comment:
We are here at the triple level, i.e. `describedby` is part of the (correct) URI `http://www.w3.org/2007/05/powder-s#describedby`. That's versus "The JSON field name" which is just an arbitrary label choosed in #358 to be camelCased as `describedBy`. This label, and _only_ this, is still there, see the JSON test files.